packages/workflow/src/index.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * WordPress dependencies
+ */
+import { createRoot, createElement } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { WorkflowMenu } from './components/workflow-menu';
+
+// Initialize the workflow palette if the experiment is enabled
+if ( window.wpWorkflowPaletteEnabled ) {
+	const root = document.createElement( 'div' );
+	document.body.appendChild( root );
+
+	if ( createRoot ) {
```

Comment:
It feels like this check is not unecessary,  createRoot is imported and should always be available.